.php-cs-fixer.php

```diff
@@ -8,6 +8,7 @@
 $config = new PhpCsFixer\Config();
 return $config
     ->setRules([
```

Comment:
With risky rules we need to explicitly allow them:
```suggestion
    ->setRiskyAllowed(true)
    ->setRules([
```